core/merginapi.cpp

```diff
@@ -1474,6 +1468,21 @@ ProjectDiff MerginApi::localProjectChanges( const QString &projectDir )
   return compareProjectFiles( projectMetadata.files, projectMetadata.files, localFiles, projectDir, config.isValid, config );
 }
 
+bool MerginApi::parseVersion( const QString &version, int &major, int &minor )
+{
+  QRegularExpression re;
+  re.setPattern( QStringLiteral( "(?<major>\\d+)[.](?<minor>\\d+)" ) );
+  QRegularExpressionMatch match = re.match( version );
+  if ( match.hasMatch() )
+  {
+    major = match.captured( "major" ).toInt();
+    minor = match.captured( "minor" ).toInt();
+    return true;
+  }
+
+  return false;
```

Comment:
instead of return false/true (that is not used anyways in the parent scope), can we just log `version` to `coreutils:log` with some appropriate message?

Reply:
Currently used return value